lib/tab.js

```diff
@@ -167,9 +168,9 @@ class Tab {
             return;
         }
 
-        let folder = this.pane.getPath().split('/');
+        let folder = this.pane.getPath().split(pathSep);
         const numOfFolders = atom.config.get('tab-foldername-index.numberOfFolders');
-        folder = folder.slice(folder.length - numOfFolders - 1, -1).join('/');
+        folder = folder.slice(folder.length - numOfFolders - 1, -1).join(pathSep);
```

Comment:
I just realized, the display looks weird with the backslash so maybe this should be configurable or just rendered as a regular slash